MVC/models/articles.model.js

```diff
@@ -1,15 +1,49 @@
 const db = require("../../db/connection.js");
 const format = require("pg-format");
+const { sort } = require("../../db/data/test-data/articles.js");
 
 const nonExistentArticleCustomError = {
   status: 410,
   message: "??? There isn't an article stored in this parameter.",
 };
 
-const fetchArticlesAndTotalComments = () => {
-  return db.query(`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS total_comment_count FROM articles 
- LEFT JOIN comments ON comments.article_id = articles.article_id
- GROUP BY articles.article_id ORDER BY articles.created_at DESC;`);
+const fetchArticlesAndTotalComments = (queries) => {
+  let queryString = `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS total_comment_count FROM articles 
+  LEFT JOIN comments ON comments.article_id = articles.article_id
+  GROUP BY articles.article_id`;
+
+  // let validQueries = [
+  //   "article_id",
+  //   "title",
+  //   "topic",
+  //   "author",
+  //   "created_at",
+  //   "votes",
+  //   "article_img_url",
+  //   "total_comment_count",
+  // ];
+
+  let validOrderBys = ["asc", "desc"];
+
+  let queryValues = []; // queryValues array is useless by the way. it's just a fun reference
```

Comment:
if this is useless lets remove it then we don't want to keep anything that makes the code more confusing